Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -67,7 +68,7 @@ public void onCreate() {
             mTagsBucket = mSimperium.bucket(tagSchema);
             mPreferencesBucket = mSimperium.bucket(new Preferences.Schema());
             mPreferencesBucket.start();
-
+            
```

Comment:
```suggestion

```

This looks like some unnecessary spaces might have been accidentally added.  We can remove them.